.vs/tasks.vs.json

```diff
@@ -0,0 +1,65 @@
+{
+	"version": "0.2.1",
+	"tasks": [
+		// PYTHON ENVIRONMENT
+		{
+			"taskName": "pipenv_install",
+			"taskLabel": "setup python environment",
+			"appliesTo": "requirements.txt",
+			"type": "launch",
+			"contextType": "custom",
+			"command": "pipenv",
+			"args": [
+				"install",
+				"-r",
+				"requirements.txt"
+			],
+			"env": {
+				"PIPENV_NOSPIN": 1,
+				"PIPENV_VENV_IN_PROJECT": 1
+			},
+			"output": "HELL YEAH"
+		},
+		// CONAN INSTALL TASKS
+		{
+			"taskName": "conan_install-x86_64-Debug",
+			"taskLabel": "install Conan dependencies (x86_64-Debug)",
+			"appliesTo": "conanfile.py",
+			"type": "launch",
+			"contextType": "custom",
+			"command": "pipenv",
+			"args": [
+				"run",
+				"conan",
+				"install",
+				".",
+				"-if",
+				"${workspaceRoot}\\out\\build\\x86_64-Debug",
+				"-s",
+				"compiler.cppstd=17",
+				"--profile",
+				"x86_64-Windows-v142-Debug"
```

Comment:
We can not use this profile because it is not available for foreign developers. We should use command-line arguments or (possibly, better) commit this profiles in repo and use them in commands.

Reply:
Right, I will create profile. Should we support 32 bits platform?

Reply:
Yeah, because at least bootstrapper should be runnable on x86 (but it is not required for ALL binaries, for example CLI).